app/src/main/java/nl/nfi/cellscanner/Database.java

```diff
@@ -273,6 +274,19 @@ public void storePhoneID(Context ctx) {
         setMetaEntry("android_id", android_id);
     }
 
+    public void storeLocationInfo(Location location) {
+        ContentValues values = new ContentValues();
+        values.put("provider", location.getProvider());
+        values.put("timestamp", location.getTime());
+        values.put("accuracy", location.getAccuracy());
+        values.put("latitude", location.getLatitude());
+        values.put("longitude", location.getLongitude());
```

Comment:
altitude?

Reply:
Willen we dat?

https://developer.android.com/reference/android/location/Location.html

Kijk gelijk ff wat we allemaal willenen en hoevaak =D

Reply:
Altitude zou misschien nuttig kunnen zijn, bijvoorbeeld in hoge gebouwen, al weet ik dat het bij GPS niet bijzonder nauwkeurig is.

Alle andere dingen, zoals bearing, speed etc, kan ik geen toepassing voor bedenken

Reply:
Top, ga ik dat toevoegen

Reply:
Zou speed niet uiteindelijk gebruikt kunnen worden om te voorspellen of iemand in een voertuig zat of niet? (gezien de meeting die daarover binnenkort is)

Reply:
done